README.md

```diff
@@ -33,18 +33,28 @@ You can generate a PDF or an HTML copy of this guide using
 
 ## Table of Contents
 
-* [Source Code Layout & Organization](#source-code-layout--organization)
-* [Syntax](#syntax)
-* [Naming](#naming)
-* [Strings](#strings)
-* [Macros](#macros)
-    * [Macro Declarations](#macro-declarations)
-* [Anonymous (lambda) functions](#anonymous-lambda-functions)
-* [Loading and Autoloading](#loading-and-autoloading)
-* [Comments](#comments)
-    * [Comment Annotations](#comment-annotations)
-    * [Docstrings](#docstrings)
-* [Existential](#existential)
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-generate-toc again -->
+**Table of Contents**
+
+- [The Emacs Lisp Style Guide](#the-emacs-lisp-style-guide)
```

Comment:
:man_shrugging: I'll remove it, but not the TOC was incomplete when I ran `markdown-toc`, so I think the manual solution is not that reliable.